makefile

```diff
@@ -7,23 +7,47 @@ SFILES      = *.s
 SRCDIR      = src
 OBJDIR      = obj
 BINDIR      = bin
-VC_VERSIONS = D43J NACJ NACE NARJ NARE
+VC_VERSIONS = PZLJ PZLE
 NAME        = homeboy
 RESDESC     = res.json
 
+# either 'OOT' or 'MM'
+GAME ?= OOT
+
+ifeq ($(GAME),)
+$(error "GAME unset.")
+else ifneq ($(GAME),OOT)
+ifneq ($(GAME),MM)
+$(error "Wrong value for GAME.")
+endif
+endif
```

Comment:
maybe something like this would be prettier:
```
ifeq ($(GAME),OOT)
VC_VERSIONS = ... PZLJ PZLE ...
else ifeq ($(GAME),MM)
VC_VERSIONS = ... PZLJ PZLE ...
else
$(error ...)
endif
```